packages/ui/components/form/dropdown/Dropdown.tsx

```diff
@@ -137,7 +137,7 @@ export const DropdownItem = (props: DropdownItemProps) => {
     <ButtonOrLink
       {...rest}
       className={classNames(
-        "hover:text-emphasis text-default inline-flex w-full items-center space-x-2 px-3 py-2  disabled:cursor-not-allowed",
+        "hover:text-emphasis text-default inline-flex w-full items-center space-x-2 px-3 py-2 transition disabled:cursor-not-allowed",
```

Comment:
I had to remove `transition` earlier because it was causing this flickering of blue border

https://github.com/calcom/cal.com/assets/53316345/0868b6c4-3cc8-4fce-8ef7-f0d2f31790d9

Reply:
This shouldn't happen which means there's an underlying issue with the `ButtonOrLink` component

Reply:
I think this was fixed in a recent commit because it's no longer happening.

Reply:
@hichemfantar i have sent the recording of this branch

Reply:
can we get rid of the blue outline thing alltogether?

Reply:
@Udit-takkar  resynced with main, try now

Reply:
@PeerRich waiting for confirmation by @Udit-takkar that this is fixed. Otherwise I'll issue a fix

Reply:
As you can see I'm not getting the blue border glitch.

https://github.com/calcom/cal.com/assets/34947993/14a717ed-c9ef-4217-9ed9-10587f5cefb3